src/scripts/validators/validators.js

```diff
@@ -0,0 +1,133 @@
+// src/scripts/validators/validators.js
+export function validateAge(age) {
+  if (age == null || age === '') {
+    return {
+      title: 'Age',
+      message: 'Age is required.',
+    };
+  }
+
+  const num = Number(age);
+
+  if (Number.isNaN(num) || !Number.isFinite(num)) {
+    return {
+      title: 'Age',
+      message: 'Age must be a valid number.',
+    };
+  }
+
+  if (!Number.isInteger(num)) {
+    return {
+      title: 'Age',
+      message: 'Age must be an integer.',
+    };
+  }
+
+  if (num < 18 || num > 90) {
```

Comment:
Consider to record this not like magic numbers but instead use predifined constants and put them here. The same for other places in code.
```
const MIN_AGE = 18
const MAX_AGE = 90
```